docs/getting_started_on_windows.md

```diff
@@ -134,7 +134,7 @@ environment variable is set when you are building.
 
 ```shell
 # Disable stupid^H^H^H^H^H^H Windows line ending translation
-git config --global core.autocrlf false
+git config --global core.autocrlf true
```

Comment:
Personally, I think this should really be a separate PR with an explanation of what's going on here, but I guess Ben did ask you to throw this in and merging via copybara is still pretty painful, which works against small PRs :-/

Ah well